src/components/character/character.jsx

```diff
@@ -53,19 +53,18 @@ const Character = React.createClass(({
         }
 
         newCharacter.guid = generateGuid();
-
+        //fieldsToSave not technically an array so we call the array method and apply the fields as though they were elements in an array
```

Comment:
Hey dude, what is the expression `[].forEach.call(fieldsToSave, saveField)` doing? Doesn't the function passed to `call` get executed 0 times (because `[]` has a length of 0)?